yarn.lock

```diff
@@ -4050,9 +4050,9 @@
   integrity sha512-W6CLUJ2eBMw3Rec70qrsEW0jOm/3twwJv21mrmj2yORiaVmVYGS4sSS5yUwvQc1ZlDLYGPnClVWmUUMagKNsfA==
 
 "@microsoft/microsoft-graph-types@^2.6.0":
-  version "2.13.0"
```

Comment:
not sure what these `yarn.lock` changes are, can we revert to what's in main?

Reply:
Sure 👍🏻

Reply:
@benjdlambert  Done ✌🏼